plenigo_i18n/plenigo-de_DE.po

```diff
@@ -1,688 +1,658 @@
+msgstr "Vorname:"
 
-#: plenigo_plugin/PlenigoLoginWidget.php:120
-msgid "Welcome, "
-msgstr "Willkommen"
+#: ../plenigo_plugin/PlenigoShortcodeManager.php:396
+msgid "Last Name:"
+msgstr "Nachname:"
 
-#: plenigo_plugin/PlenigoLoginWidget.php:121
-#: plenigo_plugin/PlenigoLoginWidget.php:161
-msgid "Edit Profile"
-msgstr "Profil bearbeiten"
+#: ../plenigo_plugin/PlenigoShortcodeManager.php:397
+msgid "Street Address:"
+msgstr "Straße und Nummer:"
 
-#: plenigo_plugin/PlenigoLoginWidget.php:122
-#: plenigo_plugin/PlenigoLoginWidget.php:165
-msgid "Logout"
-msgstr "Logout"
+#: ../plenigo_plugin/PlenigoShortcodeManager.php:398
+msgid "Additional Information:"
+msgstr "Zusätzinformationen:"
 
-#: plenigo_plugin/PlenigoLoginWidget.php:125
-msgid "Title:"
-msgstr "Titel:"
+#: ../plenigo_plugin/PlenigoShortcodeManager.php:399
+msgid "ZIP Code:"
+msgstr "PLZ:"
 
-#: plenigo_plugin/PlenigoLoginWidget.php:157
-msgid "Welcome, <name>"
-msgstr "Willkommen, <name>"
+#: ../plenigo_plugin/PlenigoShortcodeManager.php:400
+msgid "City:"
+msgstr "Ort:"
 
-#: plenigo_plugin/PlenigoSettingsPage.php:181
-msgid "Plenigo Help"
-msgstr "Plenigo Hilfe"
+#: ../plenigo_plugin/PlenigoShortcodeManager.php:401
+msgid "Country:"
+msgstr "Land:"
+
+#: ../plenigo_plugin/PlenigoShortcodeManager.php:402
+msgid "Edit this profile at plenigo"
+msgstr "Profil bei plenigo bearbeiten"
+
+#. Base ID
+#: ../plenigo_plugin/PlenigoLoginWidget.php:55
+msgid "Plenigo login"
+msgstr "plenigo Anmelden"
 
-#: plenigo_plugin/PlenigoSettingsPage.php:195
+#: ../plenigo_plugin/PlenigoLoginManager.php:293
+msgid "Mr./Mis"
+msgstr "Herr / Frau:"
+
+#: ../plenigo_plugin/PlenigoSettingsPage.php:195
 msgid "plenigo OAuth Help"
 msgstr "plenigo OAuth Hilfe"
 
-#: plenigo_plugin/PlenigoSettingsPage.php:196
+#: ../plenigo_plugin/PlenigoSettingsPage.php:196
 msgid "In order to configure plenigo OAuth Login: "
-msgstr "Konfiguration des plenigo OAuth Login: "
+msgstr "Um plenigo OAuth-Sitzung Anmelden konfigurieren:"
 
-#: plenigo_plugin/PlenigoSettingsPage.php:197
+#: ../plenigo_plugin/PlenigoSettingsPage.php:197
 msgid ""
 "1 - Add Login redirect URL to plenigo (Usually: <b>{YOUR BLOG URL}/wp-login."
 "php</b>) "
 msgstr ""
-"1 - Umleitungs-URL nach plenigo hinzufügen (Gewöhnlich: <b>{IHRE BLOG URL}/"
-"wp-login.php</b>) "
-
-#: plenigo_plugin/PlenigoSettingsPage.php:198
-#: plenigo_plugin/PlenigoSettingsPage.php:204
-msgid "clicking this link"
-msgstr "klicken Sie diesen Link"
-
-#: plenigo_plugin/PlenigoSettingsPage.php:200
-msgid "2 - Fill the same URL in the <b>OAuth redirect URL</b> below"
-msgstr "2 - Fügen Sie die selbe URL in die <b>OAuth redirect URL</b> ein"
-
-#: plenigo_plugin/PlenigoSettingsPage.php:201
-msgid ""
-"3 - (Optional) Fill the URL in the <b>URL After Login</b> for login "
-"redirection"
-msgstr ""
-"3 - (Optional) Hinterlegen Sie eine URL in <b>URL After Login</b> für eine "
-"Umleitung nach dem Login"
+"1 - In Anmelden redirect URL plenigo (In der Regel: <b>(Die URL Ihres "
```

Comment:
1 - Fügen Sie die Umleitungs-URL zu plenigo hinzu (In der Regel: <b>(Die URL Ihres